app/src/main/java/org/mozilla/focus/tiles/HomeTile.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.tiles
+
+import org.json.JSONObject
+import java.util.UUID
+
+private const val KEY_URL = "url"
+private const val KEY_TITLE = "title"
+
+private const val KEY_IMG = "img"
+private const val KEY_ID = "id"
+
+sealed class HomeTile(val url: String, val title: String) {
+    protected fun appendToJSONObject(obj: JSONObject) {
```

Comment:
I don't know if I like this `appendTo` - maybe this can create the base JSONObject with these shared key/value pairs? like super.createBaseJSON()

Reply:
Got it: I'll do `super.toJSONObject` and enter my items into that.